src/xrp/xrp_helpers.h

```diff
@@ -39,7 +39,7 @@ typedef struct {
     char buf[XRP_ADDRESS_SIZE];
 } xrp_address_t;
 
-void xrp_public_key_hash160(xrp_pubkey_t *pubkey, uint8_t *out);
+cx_err_t xrp_public_key_hash160(xrp_pubkey_t *pubkey, uint8_t *out);
```

Comment:
Why not return 0/-1 ?

Reply:
Just to keep using the error codes from the sdk, allowing a more homogeneous api usage.
Also, the called function `cx_hash_no_throw` return a `cx_err_t`, so it makes sens to propagate the error to the caller.